lutris/runners/steam.py

```diff
@@ -235,6 +239,17 @@ def get_steamapps_dirs(self):
                     dirs.append(path)
                 i += 1
 
+        # New Custom dirs
+        library_config = self.get_library_config()
+        if library_config:
```

Comment:
Can apply the walrus operator (added in Python 3.8, example [here](https://deepsource.io/blog/python-walrus-operator/)):
```suggestion
        if library_config := self.get_library_config():
```